.github/workflows/tox.yml

```diff
@@ -82,8 +82,14 @@ jobs:
     name: >-
       py${{ matrix.python-version }}@${{ matrix.os }}
     runs-on: ${{ matrix.os }}
+    continue-on-error: >-
+      ${{
+        matrix.experimental
+        && true
+        || false
+      }}
     strategy:
-      # fail-fast: false
+      fail-fast: false
```

Comment:
@ssbarnea could you explain why this change is needed?

Reply:
Not sure if needed, it was in docs example around using experimental jobs. I removed it in last patch.